main.cc

```diff
@@ -8,30 +8,6 @@
 #include <fcntl.h>
```

Comment:
looks like we need to include fcntl.h in lo.h for the O_RDONLY macro definition.